test/parallel/test-require-resolve.js

```diff
@@ -37,3 +37,4 @@ assert.strictEqual('path', require.resolve('path'));
 
 // Test configurable resolve() paths.
 require(fixtures.path('require-resolve.js'));
+require(fixtures.path('resolve-paths', 'default', 'verify-paths'));
```

Comment:
Could you add `.js` to the last arg?
(much debate about extensionless requires related to ESM, and future ESM CJS interop)